edi_exchange_template/tests/test_edi_backend_output.py

```diff
@@ -72,7 +72,7 @@ def _setup_records(cls):
         )
         cls.tmpl_out2 = model.create(
             {
-                "code": "edi.output.demo_backend.test_type_out2",
+                "code": "edi.output.output.demo_backend.test_type_out2",
```

Comment:
this should be `edi.output.generate.*` and `_get_output_template` should be fixed.
@etobella do you have tests failing here? It's all good for me locally :thinking:

Reply:
Yes, it is failing, with the latest changes, as output and input have been added to the base of the code name